MDAttributes/MDReplicated.cs

```diff
@@ -10,6 +10,7 @@ public enum MDReplicatedType
 {
     OnChange, // Replicates the value when the value has changed
     Always, // Replicates the value every frame, recommend using Unreliable
+
```

Comment:
This seems like a weird whitespace addition?

Reply:
It's actually was added by formatter automatically